server/controllers/finance/reports/financial.employee.js

```diff
@@ -71,8 +76,30 @@ async function build(req, res, next) {
       debtorAggregates : debtorOperations.aggregates,
     });
 
+    // provides the latest element of the table,
+    // as the request is ordered by date, the last line item will
+    // also be the employee's balance for the search period
+    if (options.extractEmployee) {
+      if (creditorOperations.transactions.length) {
+        data.lastCum = creditorOperations.transactions[creditorOperations.transactions.length - 1];
```

Comment:
Let's call this `lastTransaction` or `lastTxn` or something like that since it has the entire transaction and not just the cumsum.